libs/features/pages/about/src/lib/components/about-hero-section/about-hero-section.component.scss

```diff
@@ -0,0 +1,45 @@
+.about-hero-container {
+    margin: 0;
+    padding: 0;
+    opacity: 1;
+    display: flex;
+
+    @media screen and (max-width: 768px) {
+        flex-direction: column; // Change to a column layout on smaller screens
+        align-items: center; // Center align content vertically
+    }
+}
+
+.about-hero-text {
+    flex: 1;
+    letter-spacing: 0px;
+    align-items: flex-start;
+    padding-left: 40px;
+    text-align: left;
+    color: #000000;
```

Comment:
For reuse of colors, use CSS variables specified in elements/theming/_variables.scss for colors/background colors